presto-benchmark/src/main/java/com/facebook/presto/benchmark/HandTpchQuery6.java

```diff
@@ -64,7 +64,11 @@ public HandTpchQuery6(LocalQueryRunner localQueryRunner)
         //    and quantity < 24;
         OperatorFactory tableScanOperator = createTableScanOperator(0, new PlanNodeId("test"), "lineitem", "extendedprice", "discount", "shipdate", "quantity");
 
-        FilterAndProjectOperator.FilterAndProjectOperatorFactory tpchQuery6Operator = new FilterAndProjectOperator.FilterAndProjectOperatorFactory(1, new PlanNodeId("test"), () -> new TpchQuery6Processor(), ImmutableList.of(DOUBLE));
+        FilterAndProjectOperator.FilterAndProjectOperatorFactory tpchQuery6Operator = new FilterAndProjectOperator.FilterAndProjectOperatorFactory(
```

Comment:
Many changes in this commit do not match what commit message says.
Please split it into two.
I think the new one should be named sth like: "Always use dictionary processing in PageProcessor"